src/ophyd_async/plan_stubs/fly.py

```diff
@@ -32,6 +32,10 @@ def time_resolved_fly_and_collect_with_static_seq_table(
     stages/unstages the devices, and opens and closes the run.
 
     """
+    # Ensure there is at least on detector
+    if len(detectors) == 0:
```

Comment:
Can also 

`if detectors:` as a list's truthiness is whether it is empty or not, and it catches `detectors = None`

Reply:
OOOh nice